api/cedar_metadata_templates/views.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import unicode_literals
+
+from rest_framework import generics
+from rest_framework import permissions as drf_permissions
+from rest_framework.exceptions import NotFound
+
+from api.base.filters import ListFilterMixin
+# from api.base.parsers import (
+#     JSONAPIMultipleRelationshipsParser,
+#     JSONAPIMultipleRelationshipsParserForRegularJSON,
+# )
+from api.base.versioning import PrivateVersioning
+from api.base.views import JSONAPIBaseView
+from api.base import permissions as base_permissions
+# from api.base.utils import get_user_auth
+from api.cedar_metadata_templates.serializers import CedarMetadataTemplateSerializer
+from framework.auth.oauth_scopes import CoreScopes
+from osf.models import CedarMetadataTemplate
+
+
+class CedarMetadataTemplateList(JSONAPIBaseView, generics.ListAPIView, ListFilterMixin):
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+    )
+    required_read_scopes = [CoreScopes.ALWAYS_PUBLIC]
+    required_write_scopes = [CoreScopes.NULL]
+
+    serializer_class = CedarMetadataTemplateSerializer
+    model_class = CedarMetadataTemplate
+
+    # This view goes under the _/ namespace
+    versioning_class = PrivateVersioning
+    view_category = 'cedar-metadata-templates'
+    view_name = 'cedar-metadata-templates-list'
+
+    def get_default_queryset(self):
+        return CedarMetadataTemplate.objects.all()
+
+    def get_queryset(self):
+        return self.get_queryset_from_request()
+
+class CedarMetadataTemplateDetail(JSONAPIBaseView, generics.RetrieveAPIView):
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+    )
+    required_read_scopes = [CoreScopes.ALWAYS_PUBLIC]
+    required_write_scopes = [CoreScopes.NULL]
+
+    serializer_class = CedarMetadataTemplateSerializer
+
+    # This view goes under the _/ namespace
+    versioning_class = PrivateVersioning
+    view_category = 'cedar-metadata-templates'
+    view_name = 'cedar-metadata-templates-detail'
+
+    def get_object(self):
+        try:
+            return CedarMetadataTemplate.objects.get(_id=self.kwargs['template_id'])
+        except CedarMetadataTemplate.DoesNotExist:
+            raise NotFound
```

Comment:
Note: I had thought about catching `MultipleObjectsReturned` in addition to `DoesNotExist`, but there is no need since [`_id` is `unique`](https://github.com/CenterForOpenScience/osf.io/blob/1722bbd83cf2777d8f936b274df54bdcca384ea9/osf/models/base.py#L252) in `ObjectIDMixin`.